#Envelope to send a serialized ROS message through the mesh system
Header header
string host
string data
